__tests__/html/transcript.upDownArrow.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <script crossorigin="anonymous" src="/__dist__/testharness.js"></script>
+    <script crossorigin="anonymous" src="/__dist__/webchat-es5.js"></script>
+    <style type="text/css">
+      .content-editable-placeholder::before {
+        content: '<div contenteditable={true}>';
+      }
+    </style>
+  </head>
+  <body>
+    <div id="webchat"></div>
+    <script type="text/babel" data-presets="env,stage-3,react">
+      const {
+        ReactTestUtils: { Simulate },
+        WebChatTest: {
+          conditions,
+          createDirectLineWithTranscript,
+          createStore,
+          elements,
+          expect,
+          host,
+          pageObjects,
+          timeouts,
+          token
+        }
+      } = window;
+
+      async function expectScroll(element, eventInitDict, shouldScroll) {
+        const scrollable = elements.transcriptScrollable();
+
+        const scrollTopAtBottommost = scrollable.scrollHeight - scrollable.offsetHeight;
+
+        scrollable.scrollTop = scrollTopAtBottommost;
+
+        Simulate.keyDown(element, eventInitDict);
+
+        // "scrollTop" may not move immediately, but probably within the next 500 ms.
+        await new Promise(resolve => setTimeout(resolve, 500));
+
+        if (shouldScroll) {
+          expect(scrollable.scrollTop).toBeLessThan(scrollTopAtBottommost, `Should scroll for ${element.outerHTML}`);
+        } else {
+          expect(scrollable.scrollTop).toBe(scrollTopAtBottommost, `Should not scroll for ${element.outerHTML}`);
+        }
+      }
+
+      const NULL_FN = () => {};
+
+      (async function () {
+        const res = await fetch('assets/transcripts/markdown-message.json');
+
+        if (!res.ok) {
+          throw new Error(`Server returned ${res.status} while fetching transcript`);
+        }
+
+        const activityMiddleware = () => next => (...args) => {
+          const [{ activity }] = args;
+
+          if (activity.type === 'test-input') {
+            return (
+              <div>
+                <section className="elements-should-scroll">
+                  <h1>Elements here should scroll on arrow up/down</h1>
+                  <article>
```

Comment:
Do we need to test behavior when it's a mix of 'should scroll' and 'should not scroll'?

Reply:
For mix, do you mean <kbd>ALT</kbd> key should scroll for some elements that should not scroll normally?

Or do you mean the mix is, "if the <textarea> has or has no content, we should scroll or not scroll"?